recipes/fastq_utils/meta.yaml

```
@@ -1,25 +1,27 @@
-{% set version = "0.25.1" %}
+{% set version = "0.25.2" %}
 
 package:
   name: fastq_utils
   version: {{ version }}
 
 source:
   url: https://github.com/nunofonseca/fastq_utils/archive/{{ version }}.tar.gz 
-  sha256: ffdb8540c91b77024df5e24ed259fa923e661e3eda4bf67acba58c836c818d44 
+  sha256: e37b7c0a92ab2d0f45e5135c94088f4dfeee6cd371e9c78b3628351b77314479
 
 build:
-  number: 1
+  skip: True  # [win or osx]
+  number: 0
 
 requirements:
   build:
     - {{ compiler('c') }}
     - make
     - zlib
   host:
-    - samtools 0.1.19=h270b39a_9
     - zlib
     - ncurses
+    - samtools 0.1.19 = h20b1175_10 # [linux]
+#    - samtools 0.1.19 = hb42cafa_10 # [osx]
```

minor cleanup:

```suggestion
    - samtools 0.1.19 =h20b1175_10 # [linux]
```

Can you clarify though, do you need to pin to this exact build of samtools? It might limit what else can be installed into the same environment.